resources/views/searches/resources/groups/index.blade.php

```diff
@@ -1,5 +1,186 @@
+            <div class="container text-center">
+                <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+                <span id="add_success" style="display:none">{{ __('group.group_add_success') }}</span>
+                <span id="edit_success" style="display:none">{{ __('group.group_edit_success') }}</span>
+                <span id="delete_success" style="display:none">{{ __('group.group_delete_success') }}</span>
+            </div>
+        </div>
+        <div class="row text-center margin-top-bottom">
+            <div class="col-sm-2">
+                <select class="form-control" id="status-groups-filter">
+                    <option value=""> {{ __('actions.filter_by_status') }} </option>
+                    <option value="{{ __('group.status_active_value') }}"> {{ __('group.status_active') }} </option>
+                    <option value="{{ __('group.status_closed_value') }}"> {{ __('group.status_closed') }} </option>
+                </select>
+            </div>
+            <div class="col-sm-10 text-right">
+                <!-- Add group button - OPEN -->
+                @if (Auth::user()->profile != 'guest')
+                <span data-toggle="modal" href="#addGroupModal">
+                    <button type="button" class="btn btn-sm btn-outline-primary">
+                        <span class="octicon octicon-plus"></span>
+                        {{ __('actions.add') . ' ' . __('group.group') }}
+                    </button>
+                </span>
+                @endif
+                <!-- Add group button - CLOSE -->       
+            </div>
+        </div>   
+        
+        <div class="row text-center margin-top-bottom">
+            <div class="col-sm-12">
+                <table class="table dt-responsive nowrap table-hover text-center" id="groups" style="width: 100%">        
+                    <thead>
+                        <tr>
+                            <th scope="col"> {{ __('group.id') }} </th>
+                            <th scope="col"> {{ __('group.status') }} </th>
+                            <th scope="col"> {{ __('group.vehicle') }} </th>
+                            <th scope="col"> {{ __('group.broadcast') }} </th>
+                            <th scope="col"> {{ __('group.gps') }} </th>
+                            <th scope="col"> {{ __('group.people_involved') }} </th>
+                            <th scope="col"> {{ __('forms.actions') }} </th>        
+                        </tr>
+                    </thead>
+                    <tbody>
+                    </tbody>        
+                </table>
+            </div>    
+        </div>    
+    </div>
+    <!-- Content - CLOSE -->
+    @include('searches.resources.groups.add_group',['search_id' => $search->id])
+    @include('searches.resources.groups.edit_group')
+    @include('searches.resources.groups.delete_group')
+
+
```

Comment:
Missing <!-- If exists searches - CLOSE -->

Reply:
i assume you're referring to whitespace, i removed it already